### DISCUSSION TITLE
can no longer open right-click dropdown on selected notes (+ issues for just normally opening it)

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#21108] [Q&A] [answered]
https://user-images.githubusercontent.com/78441708/199818334-24cee367-eefc-4080-9b57-ab4bc1e22674.mp4

all clicks in this video are right-clicks. i am unable to open the dropdown on already selected notes, i have to open it while selecting other notes? and because of this behavior i cannot open the dropdown for multiple notes at once to combine them. 

this seems to be a similar issue to #21107, as in, related to editor selectioning.

**Comment by Axilotl17** [comment]
i suspect this is related to #20965

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/21113
